src/main/java/com/fireteam322/frc/robot/subsystems/HookArm.java

```diff
@@ -0,0 +1,14 @@
+package com.fireteam322.frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+public class HookArm extends WPI_TalonFX {

```

Comment:
I extended this to the new talo class in case we need to add extra which I doubt